examples/xor-mlp/project.clj

```diff
@@ -0,0 +1,6 @@
+(defproject learn-cortex "0.0.1-SNAPSHOT"
+  :dependencies [[org.clojure/clojure "1.8.0"]
+
+                 [thinktopic/cortex "0.9.11"]
```

Comment:
`thinktopic/experiment` includes cortex already, so this dep is unnecessary:
https://github.com/thinktopic/cortex/blob/bfaef1dbd2386b7642c8891ad765c44abeef3ee4/experiment/project.clj#L4